docs/src/Modal.doc.js

```diff
@@ -45,9 +45,9 @@ card(
       },
       {
         name: 'heading',
-        type: `string`,
+        type: `React.Node`,
```

Comment:
I get that `React.Node` includes `string`, but because we treat strings as a special type, I'd prefer if the type here was `React.Node | string`

Reply:
Makes sense and I agree - updated!